src/pages/components/combobox.mdx

```diff
@@ -0,0 +1,204 @@
+  - dropdowns/src/elements/Dropdown/Dropdown.tsx
+  - dropdowns/src/elements/Fields/Field.tsx
+  - dropdowns/src/elements/Fields/Hint.tsx
+  - dropdowns/src/elements/Menu/Items/Item.tsx
+  - dropdowns/src/elements/Fields/Label.tsx
+  - dropdowns/src/elements/Menu/Menu.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To filter down a list of options</li>
+      <li>To quickly find a known option</li>
+      <li>As a search field</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>For a searchable dropdown, use Autocomplete instead</li>
+      <li>To make more than one selection, use Multiselect instead</li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+As the user types, options appear in the list box that match what the user is typing.
+
+import ComboboxDefault from '../../examples/dropdowns/combobox/Default.tsx';
+import ComboboxDefaultCode from '!!raw-loader!../../examples/dropdowns/combobox/Default.tsx';
+
+<CodeExample code={ComboboxDefaultCode}>
+  <ComboboxDefault />
+</CodeExample>
+
+### Hidden Label
+
+The label can be hidden as needed.
+
+import ComboboxHidden from '../../examples/dropdowns/combobox/Hidden.tsx';
+import ComboboxHiddenCode from '!!raw-loader!../../examples/dropdowns/combobox/Hidden.tsx';
+
+<CodeExample code={ComboboxHiddenCode}>
+  <ComboboxHidden />
+</CodeExample>
+
+### Media
+
+You can place the magnifying glass icon at the start of the input when using the Combobox for search.
+
+import ComboboxMedia from '../../examples/dropdowns/combobox/Media.tsx';
+import ComboboxMediaCode from '!!raw-loader!../../examples/dropdowns/combobox/Media.tsx';
+
+<CodeExample code={ComboboxMediaCode}>
+  <ComboboxMedia />
+</CodeExample>
+
+### Selection Logic
+
+The combobox can can be customized to include selection logic that highlights the first relevant
+result as the user types, allowing the user to hit enter to fully select the item. If this logic
+is not included, the user must manually select a result from the list.
+
+import ComboboxSelection from '../../examples/dropdowns/combobox/Selection.tsx';
+import ComboboxSelectionCode from '!!raw-loader!../../examples/dropdowns/combobox/Selection.tsx';
+
+<CodeExample code={ComboboxSelectionCode}>
+  <ComboboxSelection />
+</CodeExample>
+
+### Size
+
+Combobox can be displayed in either default or compact size.
+
+import ComboboxSize from '../../examples/dropdowns/combobox/Size.tsx';
+import ComboboxSizeCode from '!!raw-loader!../../examples/dropdowns/combobox/Size.tsx';
+
+<CodeExample code={ComboboxSizeCode}>
+  <ComboboxSize />
+</CodeExample>
+
+## How to use it well
+
+<BestPractice>
+  <Do imageSource={props.data.comboboxHighlightDo.childFile.childImageSharp.gatsbyImageData}>
+    <p>Use balanced and appropriate typographic hierarchy.</p>
+  </Do>
+  <Dont imageSource={props.data.comboboxSearchableDont.childFile.childImageSharp.gatsbyImageData}>
+    <p>Don’t mix up typographic hierarchy when customizing item content.</p>
+  </Dont>
```

Comment:
These best practices don't seem to reflect the Figma asset.

Reply:
Yes an image seems to be displayed incorrectly, if that is what you mean - or did you have something else in mind?

Reply:
Seems like the copy for "Do This" and "Not This" might have been left over from a copy paste of `timeline.tsx`. Typographic hierarchy doesn't seem to be related to combo box:

Do This:

> When using Combobox as a search input, automatically highlight the first suggestion in the list box and allow the user to hit the Enter key to submit the search.

Not This:

> Use an Autocomplete component when creating a searchable dropdown.

Reply:
Ahh I misunderstood what was meant by assets (thinking of images). Good catch on the copy, that has been updated!